x-pack/plugins/security_solution/public/management/cypress/e2e/sentinelone/isolate.cy.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { recurse } from 'cypress-recurse';
+import { openAlertDetailsView } from '../../screens/alerts';
+import { closeAllToasts } from '../../tasks/toasts';
+import { toggleRuleOffAndOn, visitRuleAlerts } from '../../tasks/isolate';
+import { cleanupRule, loadRule } from '../../tasks/api_fixtures';
+import { ROLE, login } from '../../tasks/login';
+import { disableExpandableFlyoutAdvancedSettings } from '../../tasks/common';
+import { waitForAlertsToPopulate } from '../../tasks/alerts';
+
+// Currently it supports only Multipass, so it's not possible to test it on CI, Vagrant support will be added later.
+describe(
+  'Isolate command',
+  {
+    tags: [
+      '@ess',
+      '@serverless',
+
+      // Not supported in serverless!
+      // The `disableExpandableFlyoutAdvancedSettings()` fails because the API
+      // `internal/kibana/settings` is not accessible in serverless
+      '@brokenInServerless',
+    ],
+    env: {
+      ftrConfig: {
+        kbnServerArgs: [
+          `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+            'sentinelOneManualHostActionsEnabled',
+          ])}`,
+          `--xpack.stack_connectors.enableExperimental=${JSON.stringify([
+            'sentinelOneConnectorOn',
+          ])}`,
+          `--xpack.actions.preconfigured=${JSON.stringify({
+            'preconfigured-sentinelone': {
+              name: 'preconfigured-sentinelone',
+              actionTypeId: '.sentinelone',
+              config: {
+                url: process.env.SENTINELONE_URL,
+              },
+              secrets: {
+                token: process.env.SENTINELONE_TOKEN,
+              },
+            },
+          })}`,
+        ],
+      },
+    },
+  },
+  () => {
+    before(() => {
+      cy.dataSession(
+        'SENTINELONE_HOST', // data name
+        () => cy.task('createSentinelOneHost', {}, { timeout: 100000000 }),
```

Comment:
I take it this `timeout` value is a mistake right?

Reply:
decreased